easy-spinnaker/inst_oes.sh

```diff
@@ -0,0 +1,52 @@
+# K.Srinivas 5-Dec-2019
+#Script to be executed on node1
+#For creating PVs, PVCs and binding them
+#Not sure which PVCs need to be created, but they should be in the help package
+#################################################################################################################
+### PLEASE UPDATE THESE WITH THE USERNAME AND CREDENTIALS provided by OpsMX
+#################################################################################################################
+DOCKER_USERNAME=<username>
+DOCKER_PASSWORD=<password>
+#################################################################################################################
+
+export KUBECONFIG=/home/vagrant/.kube/config
+
+kubectl taint nodes $(hostname) node-role.kubernetes.io/master:NoSchedule-
+kubectl create ns oes
+kubectl create secret docker-registry oes-repo --docker-username=$DOCKER_USERNAME --docker-password=$DOCKER_PASSWORD --docker-email=opsmx@example.com --namespace oes
```

Comment:
This we should do. If the user does not provide the values, fail the script with proper error message. And it may be a good idea to take the inputs as command line variables as opposed to having to edit files.